tensorboard/OWNERS

```diff
@@ -0,0 +1,2 @@
+approvers:
+  - Jeffwan
```

Comment:
We should just delete this directory per:
https://github.com/kubeflow/manifests/issues/415

Reply:
Some Kfdefs has dependency on this. Let me file a clean up PR. Let's merge the work to kubeflow/kubeflow#3578

Reply:
https://github.com/kubeflow/manifests/pull/1459 @jlewi  Address #415  and I remove this folder